src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const addRowBtn = document.querySelector('.append-row');
+const addColumnBtn = document.querySelector('.append-column');
+const deleteRowBtn = document.querySelector('.remove-row ');
+const deleteColumnBtn = document.querySelector('.remove-column');
+
+const table = document.querySelector('.field');
+
+const maxCount = 10;
+const minCount = 2;
+let rowCount = table.rows.length;
+let columnCount = table.rows[0].cells.length;
```

Comment:
The same issue is present here with 'columnCount'. It's better to update 'columnCount' every time the table structure is modified.